compiler/test/__snapshots__/arrays.0f9e7d37.0.snapshot

```diff
@@ -4,7 +4,9 @@ arrays › array_access
  (type $none_=>_i32 (func (result i32)))
  (type $none_=>_none (func))
  (import \"_grainEnv\" \"mem\" (memory $0 0))
+ (import \"_grainEnv\" \"tbl\" (table $tbl 0 funcref))
```

Comment:
So, these changes are just because the binaryen optimizations are disabled, right?

Reply:
Yup yup.